### PR TITLE
Docs(input): update input docs

### PR DESCRIPTION
**Pull request by rottenpen**
- [x] docs

changes: Update input docs

### CODE REVIEW

**Comment by rottenpen** [comment]
I run `yarn website-dev` and found `import { defineComponent, ref, onMounted } from "vue"` can't work in demo.
And catch this
```
ERROR in ./website/docs/zh-CN/input.md?vue&type=script&lang=ts (./node_modules/ts-loader??ref--1!./node_modules/vue-loader/dist??ref--4-0!./website/md-loader!./website/docs/zh-CN/input.md?vue&type=script&lang=ts) 410:12
Module parse failed: 'import' and 'export' may only appear at the top level (410:12)
File was processed with these loaders:
 * ./node_modules/ts-loader/index.js
 * ./node_modules/vue-loader/dist/index.js
 * ./website/md-loader/index.js
You may need an additional loader to handle the result of these loaders.
|                 ]));
|             }
>             import { defineComponent, ref, onMounted } from "vue";
|             const democomponentExport = defineComponent({
|                 setup() {
```
How can i fix it? @JeremyWuuuuu @zazzaz Maybe my last pr has same question .

**Comment by rottenpen** [comment]
@JeremyWuuuuu @zazzaz Is there any progress?

**Comment by JeremyWuuuuu** [comment]
Please refer to #529, this is not a bug.

**Comment by rottenpen** [comment]
It's ok now.

File: website/docs/fr-FR/input.md [hunk 1]
```diff
@@ -18,13 +18,15 @@ Les modificateurs de `v-model` ne sont pas supportés.
...
 <script>
-export default {
-  data() {
+import { defineComponent, ref } from 'vue'
+export default defineComponent ({
+  setup() {
+    const input = ref('')
     return {
-      input: ''
+      input
```
Comment (JeremyWuuuuu) [inline website/docs/fr-FR/input.md:26]: Maybe using short term here is better?
```suggestion
input: ref(''),
```

File: website/docs/fr-FR/input.md [hunk 2]
```diff
@@ -331,45 +368,55 @@ Vous pouvez obtenir de l'aide ou des suggestions basées sur ce que vous entrez.
...
-    methods: {
-      querySearch(queryString, cb) {
-        var links = this.links;
-        var results = queryString ? links.filter(this.createFilter(queryString)) : links;
+import { defineComponent, ref, onMounted } from 'vue';
+export default defineComponent({
+  setup() {
+    const links = ref([]);
+    const state1 = ref("");
+    const state2 = ref("");
```
Comment (JeremyWuuuuu) [inline website/docs/fr-FR/input.md:376]: It is suggested to use single quote rather than double quote.

**Comment by JeremyWuuuuu** [comment]
Please do make them consistent by marking double quotes to single quotes in `script` tag.

**Comment by rottenpen** [comment]
It’s alright.

**Review by jw-foss — APPROVED** [review]
Neat, let's merge it

**Pull request merged**